web/src/scripts/webflow/helpers.mjs

```diff
@@ -7,6 +7,8 @@ import remarkParse from "remark-parse";
 import remarkRehype from "remark-rehype";
 import { unified } from "unified";
 
+const WAIT_AMOUNT = 20000;
```

Comment:
Why is this one 20000?

Reply:
It depends on when the counter resets. In theory, we wouldn't have to wait a full 60 seconds to send additional requests because we don't necessarily need to send 120 more requests. So, to save on our CircleCI dollars, it's probably best we keep this arbitrarily short, but an amount longer than 60 / 4 = 15 seconds. (since we only have, theoretically 4-5 requests to spare before triggering the rate limit.)